pkgs/by-name/cc/cctools/package.nix

```diff
@@ -149,6 +150,10 @@ stdenv.mkDerivation (finalAttrs: {
     for arch in arm i386 x86_64; do
       mv "$gas/libexec/as/$arch/as-$arch" "$gas/libexec/as/$arch/as"
     done
+
+    # Move libtool to its own output to allow packages to add it without pulling in all of cctools
+    moveToOutput bin/${targetPrefix}libtool "$libtool"
+    ln -s "$libtool/bin/${targetPrefix}libtool" "$out/bin/${targetPrefix}libtool"
```

Comment:
It’s up to them whether they want to update or not. This was done because a few (definitely Swift, maybe .NET) were using private derivations just to link only `libtool`. With `libtool` being available via `xcrun`, some of those derivations may be able to drop cctools completely if their build systems can find it with `xcrun --find libtool`.

Reply:
Ah, I missed the `ln`.